### PR TITLE
expose node_uname_info metric

### PR DESCRIPTION
**Pull request by paulfantom**
Expose information about operating systems running OpenShift. 

Requested as RFE-82

/cc @brancz

### CODE REVIEW

File: docs/data-collection.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cluster_feature_set reports the configured cluster feature set and
   // whether the feature set is considered supported or unsupported.
   '{__name__="cluster_feature_set"}',
+  // node_uname_info reports information about OS gathered from uname call
```
Comment (squat) [inline docs/data-collection.md]: nit: `... from the uname syscall.`

**Comment by squat** [comment]
this needs another `make generate` to update the sample-metrics docs.

**Comment by paulfantom** [comment]
regenerated

**Comment by paulfantom** [comment]
yay! One RFE done!

/woof

**Pull request merged**